hw/dv/data/sim.mk

```diff
@@ -76,6 +76,14 @@ else
 	# Compile the sw test code and generate the image.
 	${LOCK_SW_BUILD} "ninja -C ${sw_build_dir}/build-out \
 		${sw_test}_export_${sw_build_device}"
+	# Convert sw image to frame format
```

Comment:
It is not useful to do this for every test. Please add TODO to work this out later.

Reply:
good catch. Added TODO. Since it only creates once for a test, perhaps don't take too much time.